### PR TITLE
Feature/wmt 678/overview front end e2e tests

### PR DESCRIPTION
**Pull request by dconey646**

### CODE REVIEW

**Comment by caoimheb** [comment]
TODO: Add to pa11y script

File: test/e2e/view-offender-manager-overview.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const expect = require('chai').expect
+
+const workloadCapacityHelper = require('../helpers/data/aggregated-data-helper')
```
Comment (andrew-js-wright) [inline test/e2e/view-offender-manager-overview.js]: We could rename this variable to something along the lines of `dataHelper` to make it more generic.

Comment (caoimheb) [inline test/e2e/view-offender-manager-overview.js]: 👍🏻

File: test/e2e/view-offender-manager-overview.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      })
+      .then(function (results) {
+        dataHelper.selectGradeForWorkloadOwner(workloadOwnerId)
+        .then(function (gradeResult) {
+          workloadOwnerGrade = gradeResult
+        })
+      })
+  })
+
+  it('should navigate to the overview page', () => {
```
Comment (caoimheb) [inline test/e2e/view-offender-manager-overview.js]: Switch to `function () {}` notation? And above

File: test/helpers/data/aggregated-data-helper.js [hunk 3]
```diff
@@ -222,6 +223,17 @@ module.exports.selectIdsForWorkloadOwner = function () {
   return promise
 }
 
+module.exports.selectGradeForWorkloadOwner = function (workloadOwnerId) {
+  var promise = knex('workload_owner')
+    .first('grade_code')
```
Comment (caoimheb) [inline test/helpers/data/aggregated-data-helper.js]: Worth calling out explicitly which table `grade_code` comes from?

File: app/services/data/get-organisational-hierarchy-data.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ module.exports = function () {
             'ldu.description AS ldu_description',
             'team.id AS team_id',
             'team.description AS team_description',
-            'offender_manager.id AS offender_manager_id',
+            'workload_owner.id AS workload_owner_id',
```
Comment (caoimheb) [inline app/services/data/get-organisational-hierarchy-data.js]: 👍🏻

**Pull request merged**